### PR TITLE
Save mzmine version to batch file and log if versions mismatch

### PR DESCRIPTION
**Pull request by robinschmid**
Show dialog when importing a batch to the batch dialog

image

Added an attribute to the batch element
<?xml version="1.0" encoding="UTF-8"?><batch mzmine_version="3.3.19">

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java [hunk 1]
```diff
@@ -52,6 +54,23 @@ public class BatchQueue extends ArrayObservableList<MZmineProcessingStep<MZmineP
 
   // Method element name.
   private static final String METHOD_ELEMENT = "method";
+  // attr of the main xmlElement
+  public static final String XML_MZMINE_VERSION_ATTR = "mzmine_version";
+
+  // added in MZmine 3.4.0 - null before that
+  @Nullable
+  private final Semver mzmineVersionCreatedIn;
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java:62]: What happens if the user corrects the values (if there are changes)? There's no way of updating the value to the current version. the error will be there forever.

Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java:62]: Oh if you import and just dismiss the warning and save the batch again the batch is in the newest version

Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/batchmode/BatchQueue.java:62]: in saving its always saving the latest version.

**Comment by robinschmid** [comment]
Closed in favor of #1094

**Pull request closed without merging**